### PR TITLE
Merge changes from previous work on this topic

### PR DESCRIPTION
**Pull request by ghogen**
This change restores some relatively minor edits to the introduction to a Docker Tools topic.

### CODE REVIEW

**Review by scottaddie — APPROVED** [review]

**Review by gewarren — APPROVED** [review]

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md [hunk 1]
```diff
@@ -19,42 +21,43 @@ With the latest versions of Docker for Windows, it is easier than ever to develo
...
 
 ![Turning on Docker Solution support in a Visual Studio 2017 project](./media/image32.png)
 
 Figure 4-26: Turning on Docker Solution support in a Visual Studio 2017 project
 
-With Visual Studio version 15.8 and later, if you have multiple projects in a solution that each have an associated container, you should consider adding Docker Compose functionality.  To add this functionality, right-click on the solution or project node, and choose **Add > Container Orchestration Support**.
+ Container orchestration support, via Docker Compose, is added by default in Visual Studio 2017 versions 15.7 or earlier. Container orchestration support is an opt-in feature in Visual Studio 2017 versions 15.8 or later, in which case Docker Compose and Service Fabric are supported.
+
+With Visual Studio version 15.8 and later, you can add support for multiple projects in a solution that each have an associated container. Right-click on the solution or project node in **Solution Explorer**, and choose **Add** > **Container Orchestration Support**.  Then choose **Docker Compose** or **Service Fabric** to manage the containers.
```
Comment (gewarren) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md]: @ghogen Do we need to add **Service Fabric** back in?

File: docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md [hunk 2]
```diff
@@ -40,22 +40,26 @@ Figure 4-27: Enable Docker support during project creation in Visual Studio 2017
...
 
-### Add container orchestrator support
+### Add container orchestration support
 
-When you want to compose a multicontainer solution, add container orchestrator support to your projects. When you add container orchestrator support, Visual Studio adds a *Dockerfile* to the project (if it doesn't already exist) and a global *docker-compose.yml* file at the solution level. This lets you run and debug a group of containers (a whole solution) at the same time if they're defined in the same *docker-compose.yml* file. If *docker-compose.yml* already exists, Visual Studio just adds the required lines of configuration code to it.
+When you want to compose a multicontainer solution, add container orchestration support to your projects. This lets you run and debug a group of containers (a whole solution) at the same time if they're defined in the same *docker-compose.yml* file.
+
+To add container orchestration support, right-click on the solution or project node in **Solution Explorer**, and choose **Add** > **Container Orchestration Support**.  Then choose **Docker Compose** or **Service Fabric** to manage the containers.
```
Comment (scottaddie) [inline docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/visual-studio-tools-for-docker.md]: Remove the extra space before "Then choose".

**Comment by rpetrusha** [comment]
You can merge this if you're ready to, @ghogen.

**Comment by rpetrusha** [comment]
Sure, I'll merge now, @ghogen.

**Pull request merged**